packages/ERTP/test/unitTests/core/test-makeMint.js

```diff
@@ -0,0 +1,119 @@
+import { test } from 'tape-promise/tape';
```

Comment:
Sorry forgot this comment. There's already a test-mint.js for testing this file. Can we move these tests to that file to avoid duplicate files?

Reply:
I wasn't sure. I was under the impression that `test-mint` was about testing methods of the mint object. It seemed different enough from testing the `makeMint` function itself, so i created a different file, but i can move the tests to the same file

Reply:
Ok, thanks! Right now, for the most part, we're making the test structure match the file structure so that would help.

Reply:
I moved the tests to test-mint.js